### PR TITLE
Feature/total item quantity

### PR DESCRIPTION
**Pull request by ghost**
#### What is the purpose of this pull request?
Added a new prop `showTotalItemsQty` to the minicart. If this prop is `true` the total quantity of items will be displayed, while if the prop is `false` the quantity of *different* items will be displayed.

Example: If you buy 2 identical pizzas, with this new prop as `true`the number `2` will appear near the cart, but if the prop is `false` the number `1` will appear instead.

#### What problem is this solving?
Some stores prefer to show the total quantity of items in the cart and this wasn't possible before this feature.

#### How should this be manually tested?
Render a Minicart with the `showTotalItemsQty` set to `true` then add more than one of the same item to the cart and watch the number that will be displayed.

#### Screenshots or example usage

Screen Shot 2019-07-16 at 11 27 12 AM

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__tests__/MiniCart.test.js [hunk 1]
```diff
@@ -125,4 +125,34 @@ describe('<MiniCart /> component', () => {
...
+
+  it('should show the quantity of different items in cart', async () => {
+    const {getByTitle} = render(
+      <MockedProvider mocks={mocks}>
+        <MiniCart type="sidebar" hideContent={false} showTotalItemsQty={true}/>
+      </MockedProvider>
+    )
+    await resolveApolloQueries()
+    expect(getByTitle('item-qty').textContent).toBe('2')
+
```
Comment (klzns) [inline react/__tests__/MiniCart.test.js]:
```suggestion
```

File: react/__tests__/MiniCart.test.js [hunk 2]
```diff
@@ -125,4 +125,34 @@ describe('<MiniCart /> component', () => {
...
+  it('should show the quantity of different items in cart', async () => {
+    const {getByTitle} = render(
+      <MockedProvider mocks={mocks}>
+        <MiniCart type="sidebar" hideContent={false} showTotalItemsQty={false}/>
+      </MockedProvider>
+    )
+    await resolveApolloQueries()
+    expect(getByTitle('item-qty').textContent).toBe('1')
+
```
Comment (klzns) [inline react/__tests__/MiniCart.test.js]:
```suggestion
```

File: react/index.js [hunk 3]
```diff
@@ -355,6 +357,7 @@ class MiniCart extends Component {
                 <IconCart size={iconSize} />
                 {quantity > 0 && (
                   <span
+                    title="item-qty"
```
Comment (klzns) [inline react/index.js]: Use `data-testid` instead

File: react/__tests__/MiniCart.test.js [hunk 4]
```diff
@@ -125,4 +125,34 @@ describe('<MiniCart /> component', () => {
       expect.objectContaining({ title: expect.any(String) })
     )
   })
+
+  it('should not show item quantity if there are no items in cart', () => {
+    const {queryByTitle} = render(
+      <MiniCart type="sidebar" hideContent={false} showTotalItemsQty={false}/>
+    )
+    expect(queryByTitle('item-qnt')).toBeNull()
```
Comment (klzns) [inline react/__tests__/MiniCart.test.js]:
```suggestion
    expect(queryByTitle('item-qty')).toBeNull()
```

File: react/index.js [hunk 5]
```diff
@@ -338,7 +340,7 @@ class MiniCart extends Component {
         pl4: quantity <= 0,
       }
     )
-
+
```
Comment (klzns) [inline react/index.js]:
```suggestion

```

**Review by klzns — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.22.0] - 2019-07-16
```
Comment (lbebber) [inline CHANGELOG.md]:
```suggestion
## [2.22.0] - 2019-07-16
```

No need for the version tag, our release process adds that automatically

**Review by klzns — APPROVED** [review]

**Review by jgfidelis — APPROVED** [review]

**Pull request merged**